flytekit/clis/sdk_in_container/package.py

```diff
@@ -50,8 +53,26 @@
     is_flag=True,
     default=False,
     required=False,
-    help="This flag enables fast packaging, that allows `no container build` deploys of flyte workflows and tasks. "
-    "Note this needs additional configuration, refer to the docs.",
+    help="[Will be deprecated, see --copy] This flag enables fast packaging, that allows `no container build`"
+    " deploys of flyte workflows and tasks. You can specify --copy all/auto instead"
+    " Note this needs additional configuration, refer to the docs.",
+)
+@click.option(
+    "--copy",
+    required=False,
+    type=click.Choice(["all", "auto", "none"], case_sensitive=False),
+    default=None,  # this will be changed to "none" after removing fast option
+    callback=parse_copy,
+    help="[Beta] Specify how and whether to use fast register"
+    " 'all' will behave as the current fast flag copying all files, 'auto' copies only loaded Python modules",
+)
+@click.option(
+    "--ls-files",
```

Comment:
From our call, we wanted to remove `--ls-files` and print all the files when `--verbose` is passed in.

Reply:
removing the flag, but it's helpful to have the option in the fastpackageoptions object so leaving it there unless there are objections.